.idea/.name

```diff
@@ -1 +1 @@
-sample-respoke-android
+sample-respoke-android
```

Comment:
Is your editor removing EOF newlines? Tisk, tisk, that's not POSIX compliant! :)